packages/hardhat-viem/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nomicfoundation/hardhat-viem",
-  "version": "1.0.4",
+  "version": "2.0.0",
```

Comment:
This is fine as long as it is for testing, but shouldn't be merged with the version bump, as `changeset` should take care of that.